shepherd/Codegen.py

```diff
@@ -0,0 +1,368 @@
+import random
+
+class Codegen:
+
+    def __init__(self, rfids, rand_seed=None):
```

Comment:
This might be mentioned somewhere, but do you have requirements on how long/large the RFIDs taken in should be.